scss/_patterns_navigation.scss

```diff
@@ -21,67 +7,67 @@
     position: relative;
     width: 100%;
 
-    &__toggle--open {
-      @extend %navigation-toggle;
+    @media (min-width: $breakpoint-medium) {
+      display: flex;
+    }
+
+    &__toggle--open,
+    &__toggle--close,
+    &__link,
+    &__link>a {
+      &:visited {
+        color: $color-light;
+      }
+    }
+
+    &__link,
+    &__link>a {
```

Comment:
Also here - Can you add a space before and after the `>` please just to keep with rest of code style conventions.